africanlii/templates/africanlii/_ratification_table.html

```diff
@@ -1,8 +1,9 @@
 {% load peachjam i18n %}
+<h2>{% trans 'Ratified documents' %}</h2>
 <table class="table">
   <thead>
     <tr>
-      <th>{% trans 'Ratification Title' %}</th>
+      <th>{% trans 'Title' %}</th>
       <th>{% trans 'Ratification Date' %}</th>
```

Comment:
I forgot to ask, pls also include the deposit date and signature dates, so that it matches the columns in the ratification tab of the document detail page

![image](https://github.com/laws-africa/peachjam/assets/4178542/da9cfcc8-f89f-4ce7-93ba-a0caadb5425d)